lib/src/model/result.dart

```diff
@@ -21,7 +21,7 @@ class Result {
     _voteAverage = result['vote_average'].toString();
     _title = result['title'];
     _popularity = result['popularity'];
-    _posterPath = result['poster_path'];
+    _posterPath = "https://image.tmdb.org/t/p/original" + result['poster_path'];
```

Comment:
lets get this path ina constants file please, so we can reuse and change it in just one place

Reply:
Fixed